ignition/service/queue.py

```diff
@@ -94,44 +96,47 @@ def __read_incoming_job(self, job_definition_str):
         return job_definition
 
     def __received_next_job_handler(self, job_definition_str):
-        logger.debug('Processing incoming job: {0}'.format(job_definition_str))
         try:
             job_definition = self.__read_incoming_job(job_definition_str)
         except JsonContent.ERROR_TYPE as e:
-            logger.exception('Ignoring job as an error occurred whilst attempting to read it: {0}'.format(job_definition_str))
+            logger.exception('Ignoring job as an error occurred whilst attempting to read it: {0}'.format(str(e)))
             return None
         return self.__handle_job(job_definition)
 
     def __handle_job(self, job_definition):
         requeue = False
         job_type = job_definition.get(self.JOB_TYPE_KEY, None)
+        job_id = job_definition.get(self.JOB_IDENTIFIER_KEY, None)
         if job_type is None:
-            logger.warning('Ignoring job received from queue without job_type: {0}'.format(job_definition))
+            logger.warning('Ignoring job received from queue without job_type for request id {0} and job id {1}'.format(job_definition.get('request_id'), job_id))
```

Comment:
`request_id` is not a mandatory attribute on all jobs so I don't think we should use it in the logs.

Reply:
Removed request_id